packages/manager/apps/telecom/src/app/app.controller.js

```diff
@@ -26,6 +26,13 @@ export default class TelecomAppCtrl {
     this.ovhFeatureFlipping = ovhFeatureFlipping;
 
     this.chatbotEnabled = false;
+    this.SYSTRON_FEEDBACK_INFO = {
+      part1:
+        'The text in this manager section is provided by Systran machine translation. Share your feedback',
```

Comment:
No translation required ?

Reply:
No translation is not required since the Systran will be used only to translate in English and we display this info when the user language is english.